### PR TITLE
Renovate: Split Dependency Updates between gradle and npm

### PR DESCRIPTION
**Pull request by DC2-DanielKrueger**
**Motivation**

Resolves #<issue>

**Changes**

### CODE REVIEW

File: renovate.json5 [hunk 1]
```diff
@@ -9,5 +9,208 @@
...
+            "matchPackagePatterns": [
+                "^com\\.github\\.everit-org\\.json-schema[.:]"
+            ],
+            "registryUrls": [
+                "https://jitpack.io"
+            ]
+        },
+        {
+            "groupName": "all patch dependencies",
+            "groupSlug": "all-patch",
```
Comment (vanch3d) [inline renovate.json5:49]: D o we want to keep the all-patches ?

File: renovate.json5 [hunk 2]
```diff
@@ -9,5 +9,208 @@
...
+        // GitHub Actions minor & patch
+        {
+            "matchManagers": [
+                "github-actions"
+            ],
+            "groupName": "all non-major GitHub Actions dependencies",
+            "groupSlug": "github-actions-non-major",
+            "matchUpdateTypes": [
+                "minor",
+                "patch"
```
Comment (vanch3d) [inline renovate.json5:170]: Same here. Is minor not enough?

**Review by vanch3d — APPROVED** [review]
All good to me. Non-blocking questions about keeping patches in the renovate groups

File: renovate.json5 (hunk 1 above)
Comment (DC2-DanielKrueger) [reply renovate.json5]: i split them up, I think we still want the patch dependencies that can be applied automotically

**Pull request merged**